zapisy/apps/grade/poll/models.py

```diff
@@ -133,7 +150,7 @@ def is_student_entitled_to_poll(self, student: Student) -> bool:
             return records_models.Record.objects.filter(
                 student=student,
                 group__course=self.course,
-                status=records_models.RecordStatus.ENROLLED
+                status=records_models.RecordStatus.ENROLLED,
```

Comment:
Nawet jeśli to nie psuje kodu i tylko wygląda dziwnie, to tej zmiany nie powinno być.
```suggestion
                status=records_models.RecordStatus.ENROLLED
```

Reply:
Największa korzyść jest z używania jednego lintera przez wszystkie osoby rozwijające kod, a tym jest w naszym projekcie, zgodnie z wiki, Flake8 i nie można tego unilateralnie zmienić.